packages/apps/reputation-oracle/server/src/modules/escrow-completion-tracking/escrow-completion-tracking.module.ts

```diff
@@ -5,11 +5,19 @@ import { TypeOrmModule } from '@nestjs/typeorm';
 import { EscrowCompletionTrackingEntity } from './escrow-completion-tracking.entity';
```

Comment:
Maybe we rename this module to `escrow-completion`? It works not only with `escrow-completion-tracking` entity and does some other things, so should be cleaner